app/assets/stylesheets/application.scss

```diff
@@ -113,45 +115,60 @@ h5 {
 }
 
 .div-bar {
-  background-color: #37becc;
-  color: #ffffff;
+  background-color: $primary-color;
+  color: $white;
   padding: 10px;
   margin: 0px;
   margin-top: 15px;
+  box-shadow: 5px 5px 5px $shadow-color;
+  border-top-left-radius: 5px;
+  border-top-right-radius: 5px;
 }
 
-.content {
-  background-color: #fff;
-  margin: 0;
-  padding: 7px;
+.div-bar a {
+  color: $alert-color;
 }
 
-#thumb {
-  margin-top: 10px;
-  height: 200px;
-  width: 200px;
-  border-radius: 50%;
+.content {
+  background-color: $white;
+  box-shadow: 5px 5px 5px $shadow-color;
+  margin: 0 0 30px;
+  padding: 0;
 }
 
-#mini-thumb {
+.user-index {
+  list-style: none;
   text-align: left;
-  height: 50px;
-  width: 50px;
-  border-radius: 50%;
+  margin: 0;
+  padding: 20px;
 }
 
-.user-index {
-  text-align: left;
-  display: inline-block;
+.user-name{
+  font-weight: bold;
+  font-size: 1em;
+  margin: 15px 0 0 0;
```

Comment:
Shorthand form for property `margin` should be written more concisely as `15px 0 0` instead of `15px 0 0 0`